l1-contracts/src/governance/Governance.sol

```diff
@@ -34,11 +34,11 @@ contract Governance is IGovernance {
 
   mapping(uint256 proposalId => DataStructures.Proposal) internal proposals;
   mapping(uint256 proposalId => mapping(address user => DataStructures.Ballot)) public ballots;
-  mapping(address => DataStructures.User) internal users;
+  mapping(address => User) internal users;
   mapping(uint256 withdrawalId => DataStructures.Withdrawal) internal withdrawals;
 
   DataStructures.Configuration internal configuration;
-  DataStructures.User internal total;
+  User internal total;
```

Comment:
This looks odd. User named total

Reply:
Ye, I know, kept it for minimum diff. Were expecting to use the same logic when it gets around the other snapshotting as part of the #13868 so would alter the name when I got to that.